Zoo/src/main/java/Zoo/Menu.java

```diff
@@ -0,0 +1,126 @@
+package Zoo;
+
+import Zoo.Birds.Crane;
+import Zoo.Birds.Crow;
+import Zoo.Birds.Peacock;
+import Zoo.Fishes.Hammerfish;
+import Zoo.Fishes.Sawfish;
+import Zoo.Fishes.Shark;
+import Zoo.Mammals.Elephant;
+import Zoo.Mammals.Lion;
+import Zoo.Mammals.Racoon;
+
+import java.util.Scanner;
+
+public class Menu {
+    public static void userMenu(){
+        Lion bonifaci = new Lion("Bonifaci");
+        Elephant dumbo = new Elephant("Dumbo");
+        Racoon rocky = new Racoon("Rocky");
+        Hammerfish tor = new Hammerfish("Tor");
+        Sawfish saw = new Sawfish("Saw");
+        Shark kilobite = new Shark("Kilobite");
+        Crane confucious = new Crane("Confucious");
+        Crow jet = new Crow("Jet");
+        Peacock handsome = new Peacock("Handsome");
+        int choice;
+        System.out.println("What would you like to see? Choose any options to get more information about animal:");
+        do {
+            Animal.getListOfAnimal();
+            System.out.println("0. Exit");
+            Scanner scanner = new Scanner(System.in);
+            choice =scanner.nextInt();
+            scanner.nextLine();
+
+            // Block of choice
+            switch (choice){
+                case 1:
+                    bonifaci.getAnimal();
```

Comment:
Would be nice to add some common method like getAllInfo() that would combine all this info together

Reply:
Implement it. Done